src/main/java/com/tailf/jnc/YangEnumeration.java

```diff
@@ -85,7 +85,7 @@ public boolean canEqual(Object obj) {
      */
     @Override
     public boolean equals(Object obj) {
-        return canEqual(obj)
+        return obj instanceof YangEnumeration
```

Comment:
I think this goes too far in placating the tool. If `canEqual` changes for whatever reason, this code becomes invalid.